shared/contract-utils/contracts/CheckpointingHelpers.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.4.24;
+
+
+library CheckpointingHelpers {
+    uint256 private constant MAX_UINT64 = uint64(-1);
+    uint256 private constant MAX_UINT192 = uint192(-1);
+
+    string private constant ERROR_UINT64_TOO_BIG = "UINT64_NUMBER_TOO_BIG";
+    string private constant ERROR_UINT192_TOO_BIG = "UINT192_NUMBER_TOO_BIG";
+
+    function toUint64Time(uint256 _a) internal pure returns (uint64) {
```

Comment:
This comes at the expense of a bit more bytecode in the base contract, but avoids the nameclash with the already existing `Uint256Helpers.toUint64()`.

It's not really ideal... but it is what it is given our limitations with mixing libraries together.

Reply:
Yes, exactly, I guess I would have called the lib `Uint64Helpers` and then have another one for `uint192` and so on, as now modifying `aragonOS` one is not an agile process...